user-interface/src/lib/components/uswds/Checkbox.tsx

```diff
@@ -60,16 +58,18 @@ const CheckboxComponent = (props: CheckboxProps, ref: React.Ref<CheckboxRef>) =>
   const checkboxTestId = `checkbox-${props.id}`;
   const labelTestId = `checkbox-label-${props.id}`;
   return (
-    <div className={classes}>
+    <div className={`usa-form-group uswds-form usa-checkbox ${props.className ?? ''}`}>
```

Comment:
This is funny.  So in other places we moved the class stuff out from inline and set up the class name outside of the jsx.  Here we did the opposite.  As long as we can test it I guess it doesn't matter.

Also, I don't think we need "uswds-form" anywhere any longer.